challenge_1/rust/deg4uss3r/challenge_1/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+pub fn string_reverse (mut in_str: std::string::String) -> std::string::String {
```

Comment:
`String` is implicitly imported as part of the prelude. You don't need to fully-qualify it, just like you don't need to `use std::string::String;` it.

Also, this should take an `&str` input parameter, since you don't need mutable access or full ownership.